awshelper/config.go

```diff
@@ -405,11 +405,35 @@ func GetAWSIdentityArn(config *AwsSessionConfig, terragruntOptions *options.Terr
 }
 
 // GetAWSUserID gets the AWS user ID of the current session configuration.
-func GetAWSUserID(config *AwsSessionConfig, terragruntOptions *options.TerragruntOptions) (string, error) {
-	identity, err := GetAWSCallerIdentity(config, terragruntOptions)
+func GetAWSUserID(sess *session.Session) (string, error) {
+	identity, err := GetAWSCallerIdentity(sess)
 	if err != nil {
 		return "", errors.New(err)
 	}
 
 	return *identity.UserId, nil
 }
+
+func ValidatePublicAccessBlock(output *s3.GetPublicAccessBlockOutput) (bool, error) {
+	if output.PublicAccessBlockConfiguration == nil {
+		return false, nil
+	}
+
+	if !aws.BoolValue(output.PublicAccessBlockConfiguration.BlockPublicAcls) {
```

Comment:
@levkohimins what happened here?

Reply:
Were you trying to test out these other configurations?

```
				PublicAccessBlockConfiguration: &s3.PublicAccessBlockConfiguration{
					BlockPublicAcls:       nil,
					BlockPublicPolicy:     nil,
					IgnorePublicAcls:      nil,
					RestrictPublicBuckets: nil,
				},
```

Reply:
Good question, not my code. In fact, it is a meaningless repetition of the same condition. Removed duplicate code, but will not improve the code in this request. I don't want regressions and delays in merging.